src/tutorials/dropbox.md

```diff
@@ -0,0 +1,72 @@
+# Setting up the Dropbox daemon
+
+- Time needed: ~10 minutes
+
+Dropbox has a very useful [Linux version](https://www.dropbox.com/install-linux) that can be installed into your Platform.sh application with a bit of intermediate [build hook](/configuration/app-containers.html#hooks) hacking.
+
+## Install the daemon
+
+Per the instructions on [installing the Linux client](https://www.dropbox.com/install-linux), you can install the client via a simple one-liner shell command, so place this into your build hook - `cd ~ && wget -O - "https://www.dropbox.com/download?plat=lnx.x86_64" | tar xzf -
+`, like so
+
+```yaml
+hooks:
+  build: |
+    echo "This is the build hook firing!"
+    cd ~ && wget -O - "https://www.dropbox.com/download?plat=lnx.x86_64" | tar
+```
+
+The build hook runs when your application's codebase is being assembled for deployment, so the filesystem is still writable.  This means that the above command will create a `~/.dropbox-dist` directory in the home directory of your project, aka "the project root", aka `/app`.  Starting the daemon is _almost_ as simple as running `~/.dropbox-dist/dropboxd` when you `ssh` into your application's container.
```

Comment:
We don't mention that `/app` is the home dir anywhere else, since that's technically an implementation detail.  Let's elide that.